config/puma.rb

```diff
@@ -0,0 +1,47 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
did we not have a puma config file before?

Reply:
We didn't. Puma became default web server in Rails 5.